docs/reference/cc-rest-api/index.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Cleura Cloud provides a REST API, along with an automatically generated documentation page.
+---
+# Cleura Cloud REST API automatically generated documentation page
+
+You may at any time make good use of the {{rest_api}} with a
+tool like `curl`. Regarding documentation, there is a detailed,
+[automatically generated page](https://apidoc.cleura.cloud),
+which you turn to for advice on endpoints and how to use them.
+
+> To access the API, you need to have an
+[account in {{brand}}](../../howto/getting-started/create-account),
+and also
+[create an access token](../../howto/getting-started/accessing-cc-rest-api).
+
+There is not much you cannot do using the {{rest_api}}, with one
```

Comment:
You might not want to not avoid a double negation. :)

```suggestion
All actions exposed via the {{gui}} are also available through the {{rest_api}}. One
```
Since CCMP is just a graphical front-end to the API, "all actions" is not an exaggeration here. Everything a customer can do in CCMP has a counterpart in the API.